File: 003/c++/Main3.cpp
#include <stdio.h>
#include <string>

int main() {
  std::string name = "Thiago";
  int age = 26;
  float weight = 53.5;
  char sex = 'M';

  printf("%s\n", name.c_str());
  printf("%d\n", age);
  printf("%f\n", weight);
  printf("%c\n", sex);
}

File: 003/c#/main3.cs
using System;

namespace Main {
  class MainClass {

    static void Main() {
      char[] name = {'T','h','i','a','g','o'};
      int age = 26;
      float weight = 53.5f;
      char sex = 'M';

      Console.WriteLine("{0}", new String(name));
      Console.WriteLine("{0}", age);
      Console.WriteLine("{0}", weight);
      Console.WriteLine("{0}", sex);
    }

  }
}
